src/Components/GrantsFinalizeShow.js

```diff
@@ -159,16 +229,15 @@ export default function GrantsFinalizeShow(props) {
           <dd>{grant.purpose}</dd>
           <dt>RFP URL</dt>
           <dd>{grant.rfp_url}</dd>
+          <p className="GrantsFinalizeShow__TotalWordCount">
+            Total word count: <span>{totalWordCount}</span>
+          </p>
```

Comment:
we should probably change this to a `dt` and `dd` so it fits semantically.
```suggestion
          <dt>Total word count:</dt>
          <dd>{totalWordCount}</dd>
```
and then we can add a css class for styling adjustments

Reply:
see this first: https://github.com/Jess-White/boilerplate_rebuild_react_app/pull/178#discussion_r650417739